### ISSUE TITLE
Strange bug on small screen in IE 10 and Edge after 7.0.0 update

### ISSUE THREAD

**Issue by giv13** [FrDH/jQuery.mmenu#853]
Hi. You can see it, if open http://mmenu.frebsite.nl in IE 10 or Edge on srceen <550px, and open/close menu. I didn't have this problem with version 6.1.8.

**Comment by FrDH** [comment]
Hi, thank you for reporting this issue, I'll have to check.

**Comment by Enkrod** [comment]
With the website not loading correctly in my IE10 I do not exactly know if I have the same problem. So here is my bug-description hoping that it matches your experience:

Bug: The mmenu closing seems to close from the wrong direction on certain widths and browsers.

Observations: 
- IE and Edge only
- It happens when the browser-width is < 550px (Note that my page is only 533px wide but the browser has an additional 17px of scrollbar)
- The animation does not seem to be broken per se but just using the wrong direction to close from
- Happens only after the menu has completely opened, not if you close it before it is fully open.

**Comment by FrDH** [comment]
is this issue a duplicate of https://github.com/FrDH/jQuery.mmenu/issues/879?
